test/system/104-run.bats

```diff
@@ -218,7 +219,7 @@ teardown() {
   create_default_container
 
   pushd /etc/kernel
-  run --separate-stderr $TOOLBOX run pwd
+  run -127 --separate-stderr $TOOLBOX run pwd
```

Comment:
Bingo!  :)

Reply:
Umm... @nievesmontero why did you mark this as resolved?  Did you investigate the test failures?  I even posted the relevant bits from the test logs [above](https://github.com/containers/toolbox/pull/1301#issuecomment-1583269631).